server/lib/routes/post-metrics.js

```diff
@@ -95,6 +104,31 @@ function optionallyLogFlowEvents (req, metrics, requestReceivedTime) {
   });
 }
 
+function isValidFlowData (metrics, requestReceivedTime) {
+  if (! metrics.flowId) {
+    return false;
+  }
+
+  const age = requestReceivedTime - parseInt(metrics.flowBeginTime);
+  if (age > FLOW_ID_EXPIRY || age <= 0 || age !== age) {
```

Comment:
Sorry, force of habit, because `NaN !== NaN`. But `isNaN` is definitely the more readable way of doing that, will change.